### DISCUSSION TITLE
Producer buffer.memory

### DISCUSSION THREAD

**Discussion by avencl** [edenhill/librdkafka#3611] [Q&A]
I was trying to find librdkafka equivalent of `buffer.memory` field, but i couldn’t, is there even such a field?

If not, what would be default value for memory size which producer uses to store messages?

**Comment by ladislavmacoun** [comment]
You can limit producer memory usage with `queue.buffering.max.messages` and `queue.buffering.max.kbytes`

https://github.com/edenhill/librdkafka/blob/master/CONFIGURATION.md
